cylc/flow/scripts/cylc.py

```diff
@@ -374,12 +374,6 @@ def iter_commands():
 
     """
     for cmd, obj in sorted(COMMANDS.items()):
-        if cmd == 'cylc':
-            # don't include this command in the listing
-            continue
-        if not obj:
-            raise ValueError(f'Unrecognised command "{cmd}"')
```

Comment:
These checks are outdated, (the first dates back to when we registered each command separately and the second to when we still had Bash commands).